tests/build-images.sh

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 set -ex
 
-ver=(NATS_SERVER 2.10.7)
+ver=(NATS_SERVER 2.9.24)
```

Comment:
Why this vs 2.10.8?

Reply:
This PR is to only bump Alpine. The 2.10.8 bump happens in a subsequent PR.